midealocal/device.py

```diff
@@ -94,10 +94,10 @@ def __init__(
         self._device_name = name
         self._device_id = device_id
         self._device_type = device_type
-        self._protocol = protocol
+        self._device_protocol_version = protocol
         self._model = model
         self._subtype = subtype
-        self._protocol_version: ProtocolVersion = ProtocolVersion.V1
+        self._message_protocol_version: ProtocolVersion = ProtocolVersion.V1
```

Comment:
```suggestion
        self._message_protocol_version: ProtocolVersion = 0
```

Reply:
Will make the logic change in a different PR, this is just the rename one ;-)